packages/story-editor/src/components/panels/design/link/link.js

```diff
@@ -284,6 +327,40 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
           </Row>
         </>
       )}
+      {showRelOption && (
+        <CheckboxContainer>
+          <StyledText size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
+            {__('About this link.', 'web-stories')}
```

Comment:
```suggestion
            {__('Qualify outbound links, 'web-stories')}
```

As per https://github.com/google/web-stories-wp/issues/4440#issuecomment-956361256

Reply:
Fixed.

Reply:
I was working of the figma.